frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -98,7 +98,7 @@ export function InsightsTable({
             render: function RenderCheckbox({}, item: IndexedTrendResult) {
                 return (
                     <PHCheckbox
-                        color={colorList[item.id]}
+                        color={colorList[item.id % colorList.length]}
```

Comment:
`getChartColors` also takes in a second `numSeries` where you can pass in the number of colors you want to generate here. I think it'll be better to pass in `items.length` there and have the util function handle any modulo wrapping